src/controllers/basket.controller.ts

```diff
@@ -146,3 +162,23 @@ const deliverableItemsHaveAddressCheck = (basketResource: Basket): boolean => {
     }
     return certificateCount > 0 || certifiedCopyCount > 0;
 };
+
+
+/**
+ * displayBasketLimitError controls the presentation of a basket limit warning/error as appropriate.
+ * @return whether a basket limit error is to be displayed (<code>true</code>), or not (<code>false</code>)
+ */
+ const displayBasketLimitError = (req: Request,
+    res: Response,
+    basketLimit: BasketLimit) : boolean => {
+if (basketLimit.basketLimitState == BasketLimitState.BELOW_LIMIT) {
+    const nextPage = `${CHS_URL}`;
+logger.debug(`Basket is not full, redirecting to ${nextPage}`);
```

Comment:
I think the indentation is a bit strange in this method.

Reply:
Do you mean it should be the first one I changed from?

Reply:
Not sure about that. But the logging/implemention lines within the `if` branch here should be indented to reflect the fact they are within the branch. Same for the lines making up the `else` branch.